### PR TITLE
Christmas event changes

### PR DESCRIPTION
**Pull request by coiax**
- Admins can now spawn `/obj/structure/flora/tree/pine/xmas/presents`
directly, instead of spawning regular christmas trees and then
triggering the event.

- The Presents Under Trees event has been removed entirely, present trees are spawned directly by the landmark spawners, and Ian hat and christmas entertainment monitor are done by celebrate()

- Stumps are now seperate items, and not just a reskinning of the tree

### CODE REVIEW

File: code/modules/events/holiday/xmas.dm [hunk 1]
```diff
@@ -11,8 +11,10 @@
 	for(var/obj/structure/flora/tree/pine/xmas/xmas in world)
 		if(!(xmas.z in GLOB.station_z_levels))
 			continue
-		xmas.icon_state = "pinepresents"
-		xmas.gifts_under_tree = TRUE
+
+		new /obj/structure/flora/tree/pine/xmas/presents(get_turf(xmas))
```
Comment (Cyberboss) [inline code/modules/events/holiday/xmas.dm]: xmas.loc

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/events/holiday/xmas.dm (hunk 1 above)
Comment (coiax) [reply code/modules/events/holiday/xmas.dm]: Why?

Comment (Cyberboss) [reply code/modules/events/holiday/xmas.dm]: When you transform something you want it to appear exactly where the old one was

Comment (coiax) [reply code/modules/events/holiday/xmas.dm]: good point

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Could you move the gifts logic to the new type please and remove the var

**Pull request merged**